client/my-sites/site-settings/sso.jsx

```diff
@@ -81,6 +87,8 @@ Sso.defaultProps = {
 };
 
 Sso.propTypes = {
+	isAtomic: PropTypes.bool,
+	siteIsJetpack: PropTypes.bool,
```

Comment:
Not used anywhere now. Something we meant to use for removing the privacy section?